### ISSUE TITLE
Some inquiries about using and modifying the code

### ISSUE THREAD

**Issue by matthewmzy** [robotflow-initiative/dipgrasp#6]
Hi there. I noticed that your pipeline require heavily on the rfu filter, which not only selects physically plausible grasp pose, but also refines the input grasp poses by closing all fingers. But this process doesn't show up in your python code. I wander how can I add some additional limitations like fixing some of my joint angles to achieve a grasp using only three fingers or others, since your unity project is invisible to us. Thanks a lot! :)

**Comment by SJTUzjy** [comment]
Hi matthewmzy.

When we released our code, we chose not to require users to handle and modify the RFU program's source code. We recognize that RFU's use of C# might be too complex for our users. Therefore, we built the RFU program as an executable file, allowing users to focus solely on the grasp generation process using Python.

However, in response to your request, we have made our RFU code available in [this GitHub repository](https://github.com/ghzh26252/GraspSimLoadMesh). Hope this will help you.
